src/server/editorServices.ts

```diff
@@ -924,7 +928,7 @@ namespace ts.server {
 
         private loadTypesMap() {
             try {
-                const fileContent = this.host.readFile(this.typesMapLocation!); // TODO: GH#18217
+                const fileContent = (this.fshost || this.host).readFile(this.typesMapLocation!); // TODO: GH#18217

```

Comment:
There are more places remaining where we are still doing this.fsHost || this.host.. They all need to be this.fsHost ?